### PR TITLE
Cleanup localized infos

### PR DESCRIPTION
**Pull request by inouire**
Cleaning localized info for which we have a flag (country_hint, airport, main_station_hint)

### CODE REVIEW

File: test_data.rb [hunk 1]
```diff
@@ -207,7 +208,7 @@ def test_homonym_localized_info
     Constants::HOMONYM_STATIONS.each do |homonym_id|
       homonym_station = STATIONS_BY_ID[homonym_id]
       Constants::SUGGESTABLE_LOCALES.each do |locale|
-        assert !homonym_station["info:#{locale}"].nil?,
+        assert (!homonym_station["info:#{locale}"].nil? || homonym_station["country_hint"] == 't')
```
Comment (espadrine) [inline test_data.rb]: Shouldn't there still be a comma at the end of the line?

Comment (inouire) [inline test_data.rb]: Indeed, fixed

File: test_data.rb [hunk 2]
```diff
@@ -448,52 +443,93 @@ def test_sncf_virtual_station
...
+    end
   end
 
   def test_country_hints_and_children
-    STATIONS.select do |row|
+    SUGGESTABLE_STATIONS.select do |row|
       row['country_hint'] == 't'
     end.each do |row|
-      # TODO once info are cleaned: check that the comment does not contain country information
+      # Check that the comment does not contain country information
```
Comment (espadrine) [inline test_data.rb]: Is there a missing check here?

Comment (inouire) [inline test_data.rb]: Indeed, but it would require including some locales, which seems a bit overkill isn't it?

Comment (espadrine) [inline test_data.rb]: How big would it be?

Comment (inouire) [inline test_data.rb]: It would require bumping the locales but only the one we're interested in, and loading them with i18n.

Comment (inouire) [inline test_data.rb]: 3 translation keys (airport, countries, main station)

File: test_data.rb [hunk 3]
```diff
@@ -448,52 +443,93 @@ def test_sncf_virtual_station
...
+      refute has_localized_info?(row, MAIN_STATION_TRANSLATIONS),
+        "One or more comments for #{row['name']} (#{row["id"]}) are mentionning a main station which is not needed as this stations is flagged with main_station_hint"
+    end
+
+    # Look for station include "main station" in the comments that should be flagged with main_station_hint
+    SUGGESTABLE_STATIONS.reject do |row|
+      row['main_station_hint'] == 't' &&
+        MAIN_STATION_TRANSLATIONS.keys.all? { |locale| row["info:#{locale}"].nil? }
+    end.each do |row|
+      print '/'
```
Comment (espadrine) [inline test_data.rb]: This print statement looks like a forgotten log.

File: test_data.rb (hunk 2 above)
Comment (espadrine) [inline test_data.rb]: Ok. We can keep the TODO to be clearer.

**Comment by espadrine** [comment]
Review done.

File: test_data.rb (hunk 3 above)
Comment (inouire) [inline test_data.rb]: Indeed... Fixed.

File: test_data.rb (hunk 2 above)
Comment (inouire) [inline test_data.rb]: :ok_hand:

**Comment by espadrine** [comment]
Thanks a lot!

**Pull request merged**